icons/jsx/SpinnerSimple.jsx

```diff
@@ -8,6 +8,7 @@ function SvgSpinnerSimple(props) {
       viewBox="0 0 24 24"
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
+      aria-label="loading"
```

Comment:
These files are generated by code. If you want to specify a label on the loading icon you need to do it in your application. Paragon doesn't have localization built in yet.

Reply:
Currently if I don't provide any icon to Stateful button, it uses a default spinner. Is there a way to add aria-label to that? 
Or will we need to explicitly add icon to stateful button and then add aria-label to it?

Reply:
We could hardcode English in Paragon for now, but if you do add your label here please: https://github.com/edx/paragon/blob/master/src/StatefulButton/index.jsx#L51

Reply:
@abutterworth I made the changes you suggested ^ but this didn't fix the accessibility problem and the icon is not focusable for stateful buttons. I have taken another route to achieving the same thing. Kindly see changes to StatefulButton component.